iBridgesCli.py

```diff
@@ -168,31 +164,31 @@ def setupIRODS(config, operation):
         resource = ic.get_resource(config['iRODS']['irodsresc'])
         try:
             print(config['iRODS']['irodsresc'] + " upload capacity, free space: " + \
-                  str(round(int(ic.resource_space(resource.name) * ic.multiplier)) + 'GB'))
+                  str(round(int(ic.resource_space(resource.name) * kw.MULTIPLIER)) + 'GB'))
         except FreeSpaceNotSet:
             ic.ienv.setdefault('force_unknown_free_space', 'True')
             print(config['iRODS']['irodsresc'] + " upload capacity, free space: No  inofrmation")
     except (NoResultFound, ResourceDoesNotExist):
         print(RED+'iRODS resource does not exist: '+config['iRODS']['irodsresc']+DEFAULT)
         try:
-            print(YEL+'Checking env-file: '+ic.default_resc)
-            resource = ic.get_resource(ic.default_resc)
-            config['iRODS']['irodsresc'] = ic.default_resc
+            print(YEL+'Checking env-file: '+ic.default_resc())
+            resource = ic.get_resource(ic.default_resc())
+            config['iRODS']['irodsresc'] = ic.default_resc()
```

Comment:
`default_resc` is now a property returning an uncallable string.

Reply:
Yep, I saw. This was already the case in the current development branch: 
https://github.com/chStaiger/iBridges-Gui/blob/a2ce3d5033d0caff5521fc2c00d4b121a243c4e2/utils/IrodsConnector.py#L133-L142
Maarten is working on an update for the CLI to fix these issues. 
To save us from doing double I did not fix it here.

Reply:
Okay.